src/yamlServerInit.ts

```diff
@@ -38,9 +40,23 @@ export class YAMLServerInit {
      * Run when the client connects to the server after it is activated.
      * The server receives the root path(s) of the workspace and the client capabilities.
      */
-    this.connection.onInitialize((params: InitializeParams): InitializeResult => {
+    this.connection.onInitialize(async (params: InitializeParams): Promise<InitializeResult> => {
+      const l10nPath: string = params.initializationOptions?.l10nPath;
+      const locale: string = params.initializationOptions?.locale;
```

Comment:
The locale is already passed from VS Code to the language server as `params.locale`. I think we can read it from there instead, which means we also don't need to send it explicitly in vscode-yaml